src/routes/event/add.tsx

```diff
@@ -41,34 +49,11 @@ export default function EventAdd() {
   };
 
   useEffect(() => {
-    get(ref(database, "location-names")).then((snapshot) => {
-      if (snapshot.exists()) {
-        const locationsResponseData = snapshot.val();
-        const newLocationOptions: string[] = [];
-        Object.keys(locationsResponseData).forEach((key) => {
-          newLocationOptions.push(key);
-        });
-        setLocationOptions(newLocationOptions);
-      } else {
-        setLocationOptions([]);
-      }
-    });
+    getAllLocations();
```

Comment:
you can combine then into 1 useEffect,
techically if you have the same stuff in the deps array, its the same useEffect